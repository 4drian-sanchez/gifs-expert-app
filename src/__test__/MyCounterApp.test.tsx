import {describe, test, expect} from "vitest"
import { render } from "@testing-library/react";
import GifsExpertApp from "../gifs/GiftsExpertApp";



describe("MyCounterApp", () => {
      test("should match with the snapshot", () => {
            
            const { container } = render(<GifsExpertApp />)
            expect(container).toMatchSnapshot()
      })   
})