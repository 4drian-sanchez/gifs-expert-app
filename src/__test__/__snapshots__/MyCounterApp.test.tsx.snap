// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MyCounterApp > should match with the snapshot 1`] = `
<div>
  <header
    class="content-center"
  >
    <h1>
      Buscador de Gifs
    </h1>
    <p>
      Descubre y comparte el Gifs perfecto
    </p>
  </header>
  <div
    class="search-container"
  >
    <input
      placeholder="Buscar Gifs"
      type="text"
      value=""
    />
    <button>
      Buscar
    </button>
  </div>
  <div
    class="previous-searches"
  >
    <h2>
      Busquedas previas
    </h2>
    <ul
      class="previous-searches-list"
    />
  </div>
  <div
    class="gifs-container"
  />
</div>
`;
